org.musescore.MuseScore.yaml

```diff
@@ -28,8 +28,9 @@ finish-args:
   # Allow loading/saving files from anywhere
   # (portals don’t work yet)
   - --filesystem=host
-  # Allow other instances to see lockfiles
-  - --env=TMPDIR=/var/tmp
+  # MuseScore want access to the session manager
+  - --talk-name=org.gnome.SessionManager
+  - --talk-name=org.a11y.Bus
```

Comment:
Its fine to have a workaround here but this sounds like a Qt bug. The portal provides access to a11y itself, raw access should never be needed, so something weird is going on.

Reply:
I have removed the a11y bus, as it was a user error here.